hack/tools/vault_utils.sh

```diff
@@ -81,6 +81,9 @@ create_policy_with_plugin_path() {
         path "$2" {
         capabilities = ["create", "read", "update", "delete", "list"]
         }
+        path "$3" {
```

Comment:
the function comments are not updated:

>  Create a policy to allow access to Vault plugin path as well as
# for the path where dataset credentials resides. This is temporary
# until dataset credentials path become obselete.
# $1 - policy name
# $2 - plugin path

obselete - > obsolete 

@revit13 do you know something about "dataset credentials path become obsolete "?